README.md

```diff
@@ -1,6 +1,6 @@
 # Help with your first commit -> [discord](https://discord.gg/k5zS4XmH)
 
-﻿## Hi Kottans!&#128521;
+## Hi Kottans!&#128521;
```

Comment:
Hey @rukhlovka you added 140 lines and deleted 110, looks like too much for couple lines change ;)